src/coreclr/interpreter/compiler.cpp

```diff
@@ -2190,7 +2535,7 @@ int InterpCompiler::GenerateCode(CORINFO_METHOD_INFO* methodInfo)
             if (m_pCBB->emitState == BBStateEmitting)
                 m_pCBB->emitState = BBStateEmitted;
             // If the new bblock was already emitted, skip its instructions
-            if (pNewBB->emitState == BBStateEmitted)
+            if ((pNewBB->emitState == BBStateEmitted))
```

Comment:
Do we need these extra parens for some reason?

Reply:
No, that is a result of me removing part of the condition that I've added during the development and then got rid of it again. In fact, Linux and macOS clang fails the build due to this.